app/controllers/services_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ServicesController < ApplicationController
+  def register
+    @service = Service.create
+    render json: @service, include: :models
```

Comment:
We also need to handle the failure cases:

1. What if the service already exists and already has all models registered? (a no-op)
1. What if there's a model name collision?